roles/openshift-applier/tasks/process-template-with-params-from-vars.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Change params_from_vars into facts
+  set_fact:
+    param: "-p {{ item.key|upper }}={{ item.value }}"
```

Comment:
I'm wondering about the `upper` aspect of this. Parameters don't have to be all upper case, but they often are. Having it enforced in the code locks us into a bit of a corner. I'm thinking we should leave this up to the user to ensure any inventory variables matches what's in the template (uppercase/lowercase/mix).